portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,82 @@
+                  stack developer. I worked on the front end of several pages of the app including the home page and the tutorial selection
+                  page and on the back end of the login page. At the end of the summer pitch
+                  competition our project took the grand prize!</li>
+
+            <li> "Little Green Foot"- This project is a sort of environmental effect tracker. In this app
+                  you could mark "environmentally friendly" actions you did that day such as carpooling to work, recycling, etc.
+                  and you would earn points from those actions which the user could cash out those points in the
+                  form of gift cards and other prizes. My role on this project was the full stack developer. I worked 
+                  with the front end developer to create all the visuals for the mock up and I worked with the backend developer
+                  to create the point system algorithm.</l1>
+        </ul>
+        <h2> If You Could Travel Anywhere in the World,Where Would You Go Kayla? </h2>
+            <p> Wow,thank y'all for asking!</p>
+
+            <p> I would say Italy is my number 1, I've always wanted to go there but if I
+                had to choose somewhere else...</p>
+            <img src="/images/italy.jpg" alt=" a cool and free pic of Italy!" style="width:50% ;height:50%">
+
+            <p>My number 2 would be Jamaica! Its so beautiful and I've never been to a place with a
+               tropical climate before so it would be a first but if I cant choose this either my
+               resulting choice would be...</p>
+            <img src="/images/jamaica.jpg" alt=" a cool and free pic of Jamaica!" style="width:50% ;height:50%">
+
+            <p> My number 3 would be France! Can't go wrong with France, it's another beautiful
+                place with alot to do.</p>
+            <img src="/images/france.jpg" alt=" a cool and free pic of France!" style="width:50% ;height:50%">
+        <h2> Kayla what are you listening to right now?</h2>
+            <p> Another good question! Well since you asked...</p>
+
+            <ul>
+                <li> I love Brockhampton. Honestly anything by them is good but if your just
+                     getting into them I would recommend the Saturation 3 album. Pure gold!</li>
+
+                <li> Same thing goes for SZA but right now I have CTRL on repeat while I wait for her
+                     to drop something new. Will she do it...I'm not sure :(</li>
+
+                <li> For some one off songs that Im enjoying now I would say "Caught Up" by Majid Jordan,
+                    "What Kinda Music" by Tom Misch, and "Target On My Chest" by Brent Faiyaz are all good.</li>
+
+                <li> At the end of the day I get all of my music from the "Pollen" playlist on Spoifify so I'd check there too.</li>
+            </ul>
+      <!--<p>Click here to get a random message:</p>    
```

Comment:
Is this intended to be commented out for now?

Reply:
Yup I was just leaving it there so I can reference some of the example JavaScript they had for later.